packages/smart-contracts/test/contracts/ChainlinkConversionPath.js

```diff
@@ -0,0 +1,180 @@
+
+const ETH_address = Currency.getCurrencyHash({type: 'ETH', value:'ETH'});
+const USD_address = Currency.getCurrencyHash({type: 'ISO4217', value: 'USD' });
+const EUR_address = Currency.getCurrencyHash({type: 'ISO4217', value: 'EUR' });
+const DAI_address = Currency.getCurrencyHash({type: 'ERC20', value:'0x38cF23C52Bb4B13F051Aec09580a2dE845a7FA35'});
+let USDT_address;
+
+let conversionPathInstance;
+let USDT_Instance;
+
+contract('ChainlinkConversionPath', (accounts) => {
+  describe('updateAggregator', async () => {
+    beforeEach(async () => {
+      conversionPathInstance = await ChainlinkConversionPath.new();
+    });
+
+    it('can updateAggregator', async () => {
+      let addressAggregator = await conversionPathInstance.allAggregators(address1, address2)
+      assert.equal(addressAggregator, '0x0000000000000000000000000000000000000000', "addressAggregator must be 0x");
+
+      await conversionPathInstance.updateAggregator(address1, address2, address3);
+
+      addressAggregator = await conversionPathInstance.allAggregators(address1, address2)
+      assert.equal(addressAggregator,address3, "addressAggregator must be 0x333..");
+    }); 
+  });
+  
+  describe('updateAggregatorsList', async () => {
+    beforeEach(async () => {
+      conversionPathInstance = await ChainlinkConversionPath.new();
+    });
+    
+    it('can updateAggregatorsList', async () => {
+      let addressAggregator = await conversionPathInstance.allAggregators(address1, address2)
+      assert.equal(addressAggregator, '0x0000000000000000000000000000000000000000', "addressAggregator must be 0x");
+
+      addressAggregator = await conversionPathInstance.allAggregators(address4, address5)
+      assert.equal(addressAggregator, '0x0000000000000000000000000000000000000000', "addressAggregator must be 0x");
+
+      await conversionPathInstance.updateAggregatorsList( [address1, address4], 
+                                                          [address2, address5], 
+                                                          [address3, address6]);
+
+      addressAggregator = await conversionPathInstance.allAggregators(address1, address2)
+      assert.equal(addressAggregator,address3, "addressAggregator must be 0x333..");
+      addressAggregator = await conversionPathInstance.allAggregators(address4, address5)
+      assert.equal(addressAggregator,address6, "addressAggregator must be 0x666..");
+    }); 
+  });
+
+  describe('getRate', async () => {
+    beforeEach(async () => {
+      conversionPathInstance = await ChainlinkConversionPath.deployed();
+      USDT_Instance = await USDT_fake.deployed();
+      USDT_address = USDT_Instance.address;
+    });
+
+    describe('only fiat rates', async () => {
+      it('can get rate from EUR to USD', async () => {
+        const conversion = await conversionPathInstance.getRate.call([EUR_address, USD_address]);
+        assert.equal(conversion.rate.toString(10), '1200000000000000000', "wrong conversion");
```

Comment:
Just a comment about the use of the 3rd argument of `assert.equal`, it's optional and I think the `wrong conversion` messages everywhere are too much. It makes the test reading more complex when your unit tests are well divided and clear. For example, `it can get rate from EUR to USD => fail` would be explicit enough, I think.